frontend/src/global_styles/layout/work_packages/_full_view.sass

```diff
@@ -131,6 +131,11 @@
     #toolbar-items
       margin-left: 0
 
+@media only screen and (max-width: $breakpoint-md)
```

Comment:
Shouldn't it be $breakpoint-sm for mobile?

Reply:
I chosehad `small` at first but looking at the medium screen sizes, I decided that this would make sense earlier. Otherwise, you'll get something like this:
<img width="635" alt="Bildschirmfoto 2025-06-30 um 13 35 10" src="https://github.com/user-attachments/assets/7945350d-3520-4ff3-8641-7ebb692ad5ab" />

compared to 
<img width="633" alt="Bildschirmfoto 2025-06-30 um 13 36 29" src="https://github.com/user-attachments/assets/bfb51d14-1db0-424f-8757-ced3c19d48ab" />

I am open to changing this back however to `small` if you consider this weird, given that the other mobile breakpoints use `small`